synfig-core/src/synfig/valuenodes/valuenode_log.cpp

```diff
@@ -120,7 +122,22 @@ ValueNode_Logarithm::operator()(Time t)const
 	DEBUG_LOG("SYNFIG_DEBUG_VALUENODE_OPERATORS",
 		"%s:%d operator()\n", __FILE__, __LINE__);
 
-	Real link     = (*link_)    (t).get(Real());
+	ValueBase input_value = (*link_)(t);
+    Real link = 0;
+
+	switch (input_value.get_type())
+	{
+		case ValueBase::TYPE_REAL:
+			link = input_value.get(Real());
+			break;
+		case ValueBase::TYPE_ANGLE:
+			link = input_value.get(Angle()); // Angle → Real
```

Comment:
The conversion shouldn't be done this way.
You should use an Angle value in an `Angle::rad()` or `Angle:degree()` first, and then use `.get()` to make it a `Real` value number.
example: 

```c++
Real real_value = Angle::rad(my_angle_var).get();
```